src/test/gather_node_test.cpp

```diff
@@ -0,0 +1,189 @@
+//*****************************************************************************
+// Copyright 2021 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+#include <algorithm>
+#include <sstream>
+
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+
+#include "../dl_node.hpp"
+#include "../entry_node.hpp"
+#include "../exit_node.hpp"
+#include "../gathernodeinputhandler.hpp"
+#include "../modelconfig.hpp"
+#include "../nodeinputhandler.hpp"
+#include "../pipeline.hpp"
+#include "../pipeline_factory.hpp"
+#include "../pipelinedefinition.hpp"
+#define DEBUG
+#include <cstdio>
+
+#include <stdlib.h>
+
+#include "../dlnodesession.hpp"
+#include "../logging.hpp"
+#include "../modelinstance.hpp"
+#include "../prediction_service_utils.hpp"
+#include "../status.hpp"
+#include "test_utils.hpp"
+
+using namespace ovms;
+using namespace tensorflow;
```

Comment:
When using both namespaces, isnt it ambigious what is Status? tensorflow::Status or ovms::Status for example

Reply:
Will remove tensorflow namespaces.